### PR TITLE
Npm lib update

### PR DESCRIPTION
**Pull request by lionsdwarf**

### CODE REVIEW

File: apps/prelaunch/lib/next-mui.js [hunk 1]
```diff
@@ -0,0 +1,169 @@
+/**
```
Comment (craigspaeth) [inline apps/prelaunch/lib/next-mui.js]: FYI: This file should be deleted with the rest of the prelaunch app I got rid of in one of my last PRs.

Comment (lionsdwarf) [reply apps/prelaunch/lib/next-mui.js]: Oops--thought I rejected that file. Just removed it.

**Comment by lionsdwarf** [comment]
@craigspaeth I removed the next-mui.js file. Is this ready for merge?

**Comment by craigspaeth** [comment]
Yep! Good work, thanks!

**Pull request merged**